src/vs/workbench/api/common/extHostNotebookDocuments.ts

```diff
@@ -32,7 +60,7 @@ export class ExtHostNotebookDocuments implements extHostProtocol.ExtHostNotebook
 	$acceptModelChanged(uri: UriComponents, event: SerializableObjectWithBuffers<extHostProtocol.NotebookCellsChangedEventDto>, isDirty: boolean, newMetadata?: NotebookDocumentMetadata): void {
 		const document = this._notebooksAndEditors.getNotebookDocument(URI.revive(uri));
 		const e = document.acceptModelChanged(event.value, isDirty, newMetadata);
-		this._onDidChangeNotebookDocument.fire(deepFreeze(e));
+		this._onDidChangeNotebookDocument.fire(partialFreeze(e));
```

Comment:
I was getting this with a full freeze:

```
ERR Cannot freeze array buffer views with elements: TypeError: Cannot freeze array buffer views with elements
	at Function.freeze (<anonymous>)
	at deepFreeze (d:\Source\vscode\out\vs\base\common\objects.js:36:20)
	at ExtHostNotebookDocuments.$acceptModelChanged (d:\Source\vscode\out\vs\workbench\api\common\extHostNotebookDocuments.js:32:77)
	at RPCProtocol._doInvokeHandler (d:\Source\vscode\out\vs\workbench\services\extensions\common\rpcProtocol.js:399:27)
	at RPCProtocol._invokeHandler (d:\Source\vscode\out\vs\workbench\services\extensions\common\rpcProtocol.js:384:45)
	at RPCProtocol._receiveRequest (d:\Source\vscode\out\vs\workbench\services\extensions\common\rpcProtocol.js:311:32)
	at RPCProtocol._receiveOneMessage (d:\Source\vscode\out\vs\workbench\services\extensions\common\rpcProtocol.js:243:26)
	at d:\Source\vscode\out\vs\workbench\services\extensions\common\rpcProtocol.js:115:52
	at Listener.invoke (d:\Source\vscode\out\vs\base\common\event.js:420:27)
	at Emitter.fire (d:\Source\vscode\out\vs\base\common\event.js:559:34)**
**
```
because the output.items.data were not freezable.